release/models/system/openconfig-system-management.yang

```diff
@@ -127,6 +127,93 @@ module openconfig-system-management {
     }
   }
 
+
+  grouping system-netconf-server-config {
+    description
+      "Configuration data for the netconf server/subsystem";
+
+    leaf netconf-transport-porotocol {
+      type enumeration {
+        enum SSH_v2 {
+          description
+            "Use SSH v2 only";
+        }
+        enum TCP {
+          description
+            "Use Transmission Control Protocol (TCP) only";
+        }
+        enum TLS {
+          description
+            "Use Transport Layer Security (TLS) only";
+        }
+        enum socket {
+          description
+            "Use socket connection only"
+      }
+      default SSH_v2;
+      description
+        "Enables the NETCONF server, using the SSHv2 transport. 
```

Comment:
Incorrect description - still no enable leaf and this service should not be enabled by default

Reply:
Description corrected